runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -32,7 +32,7 @@
 
 extern "C" {
 
-#if JAVA_SPEC_VERSION >= 16
+#if (JAVA_SPEC_VERSION >= 20) || (JAVA_SPEC_VERSION == 17)
```

Comment:
This doesn't really need to change, but if you want to change it, please write
```c
#if JAVA_SPEC_VERSION >= 17
```

Reply:
Keep JAVA_SPEC_VERSION >= 16 as suggested above.